sympy/physics/units/quantities.py

```diff
@@ -232,3 +241,17 @@ def convert_to(self, other, unit_system="SI"):
     def free_symbols(self):
         """Return free symbols from quantity."""
         return set()
+
+    @property
+    def is_prefixed(self):
+        """Whether or not the quantity is prefixed. Eg. `kilogram` is prefixed, but `gram` is not."""
+        return self._is_prefixed
+
+class PhysicalConstant(Quantity):
+    """Represents a physical constant, eg. `speed_of_light` or `avogadro_constant`."""
+
+    is_commutative = True
```

Comment:
aren't these already inherited from the superclass?

Reply:
Ah yeah, they are. Fixed.